lib/ansible/modules/git.py

```diff
@@ -241,57 +241,57 @@
 EXAMPLES = '''
 - name: Git checkout
   ansible.builtin.git:
-    repo: 'https://foosball.example.org/path/to/repo.git'
-    dest: /srv/checkout
+    repo: git@github.com:ansible/ansible.git
```

Comment:
It's probably a good idea to keep the existing mix of HTTPS and SSH style repo references while updating these:

```suggestion
    repo: 'https://github.com/ansible/ansible.git'
```

Reply:
Sure! I alternated between them since there was only a single reference using SSH style initially.